src/bridger/Bridger.sol

```diff
@@ -413,6 +413,33 @@ contract Bridger is
             inputAsset = address(WETH);
         }
 
+        if (inputAsset == sUSDe) {
+            IERC20(sUSDe).safeApprove(sUSDe, amount);
+            amount = IERC4626(sUSDe).withdraw(amount, address(this), address(this));
+        }
+
+        if (inputAsset == SOLV_BTC) {
+            amount = ISftWrapRouter(SOLV_SFT_WRAP_ROUTER).createRedemption(SOLV_BTC_POOL_ID, amount);
+        }
+
+        if (inputAsset == wUSDM) {
+            IERC20(wUSDM).safeApprove(wUSDM, amount);
+            amount = IERC4626(wUSDM).withdraw(amount, address(this), address(this));
+            // 0 coin == USDC
+            // 1 coin == USDM
+            IERC20(USDM).safeApprove(usdmCurvePool, amount);
+            amount = ICurveStableSwapNG(usdmCurvePool).exchange(1, 0, amount, 0);
```

Comment:
we need to make sure we get at least the amount we are passing to the swap and revert otherwise